tests/fixtures/circulation.py

```diff
@@ -0,0 +1,245 @@
+# MA 02111-1307, USA.
+#
+# In applying this license, RERO does not
+# waive the privileges and immunities granted to it by virtue of its status
+# as an Intergovernmental Organization or submit itself to any jurisdiction.
+
+"""Common pytest fixtures and plugins."""
+
+import json
+from copy import deepcopy
+from os.path import dirname, join
+
+import mock
+import pytest
+from invenio_circulation.proxies import current_circulation
+from utils import flush_index, mock_response
+
+from rero_ils.modules.circ_policies.api import CircPoliciesSearch, CircPolicy
+from rero_ils.modules.documents.api import Document, DocumentsSearch
+from rero_ils.modules.item_types.api import ItemType, ItemTypesSearch
+from rero_ils.modules.items.api import Item, ItemsSearch
+from rero_ils.modules.libraries.api import LibrariesSearch, Library
+from rero_ils.modules.loans.api import Loan
+from rero_ils.modules.locations.api import Location, LocationsSearch
+from rero_ils.modules.mef_persons.api import MefPerson, MefPersonsSearch
+from rero_ils.modules.organisations.api import Organisation
+from rero_ils.modules.patron_types.api import PatronType, PatronTypesSearch
+from rero_ils.modules.patrons.api import Patron, PatronsSearch
+
+
+@pytest.fixture(scope="module")
+def roles(base_app, database):
+    """Create user roles."""
+    ds = base_app.extensions['invenio-accounts'].datastore
+    ds.create_role(name='patron')
+    ds.create_role(name='librarian')
+    ds.commit()
+
+
+@pytest.fixture(scope="module")
+def patron_martigny_data(data):
+    """Load Martigny patron data."""
+    return deepcopy(data.get('ptrn5'))
+
+
+@pytest.fixture(scope="function")
+def patron_martigny_data_temp(data):
+    """Load Martigny patron data as temp."""
+    return deepcopy(data.get('ptrn5'))
+
+
+@pytest.fixture(scope="module")
+def patron_martigny(
+        app,
+        roles,
+        lib_martigny,
+        patron_martigny_data):
+    """Create Martigny librarian record."""
+    ptrn = Patron.create(
+        data=patron_martigny_data,
+        delete_pid=False,
+        dbcommit=True,
+        reindex=True)
+    flush_index(PatronsSearch.Meta.index)
+    return ptrn
+
+
+@pytest.fixture(scope="module")
+@mock.patch('rero_ils.modules.patrons.api.send_reset_password_instructions')
+def patron_martigny_no_email(
+        app,
+        roles,
+        patron_type_children_martigny,
+        patron_martigny_data):
+    """Create Martigny patron record no email."""
+    ptrn = Patron.create(
+        data=patron_martigny_data,
+        delete_pid=False,
+        dbcommit=True,
+        reindex=True)
+    flush_index(PatronsSearch.Meta.index)
+    return ptrn
+
+
+@pytest.fixture(scope="module")
+@mock.patch('rero_ils.modules.patrons.api.send_reset_password_instructions')
+def librarian_martigny_no_email(
+        app,
+        roles,
+        lib_martigny,
+        patron_type_children_martigny,
+        librarian_martigny_data):
+    """Create Martigny patron record no email."""
```

Comment:
1. patron or librarian?
2. update the comment by replacing "no email" by without sending reset password instruction